pasthis/Caddyfile

```diff
@@ -0,0 +1,51 @@
+# Pasthis Caddy configuration
+# Requires no plugins
+# - https://caddyserver.com/download/linux/amd64
+# - https://caddyserver.com/download/linux/amd64/signature
+
+
+# TLS - defaults are sane
+tls admin@paste.yourdomain.net {
+        must_staple
+}
```

Comment:
Indeed, sorry I mis-paste the content of my import directives. Fixed!
It's currently running https://p.fr33tux.org/.